### PR TITLE
Implement query to get Chainlink node's configuration

### PR DESCRIPTION
**Pull request by vyzaldysanchez**
Closes: https://app.shortcut.com/chainlinklabs/story/21717/add-a-getconfiguration-query

This aims to enable a query to retrieve the node's configuration

### CODE REVIEW

**Comment by jkongie** [comment]
Given that this is displayed in the UI as the env vars, perhaps we should consider the config to return an array of objects with something that also includes the env var name. 

Thoughts?

```
ConfigItemValue = String | Number | [String] 

type ConfigItem {
  key: String!
  value: ConfigItemValue!
}

**Review by samsondav — DISMISSED** [review]
Uff this is going to be very tedious to maintain, there are already way too many places to change when we make a configuration update.

Is there a way to do this using reflection instead or even static generation?

Otherwise we're going to have to add another case to the `config_test.go`

**Comment by jkongie** [comment]
Using the approach I suggested could allow for reflection

**Comment by vyzaldysanchez** [comment]
Thanks @samsondav for your comments and @jkongie for your suggestion, I'll be working on that!

**Review by jmank88 — DISMISSED** [review]

File: core/web/resolver/log.go [hunk 1]
```diff
@@ -26,6 +26,21 @@ func FromLogLevel(logLvl LogLevel) string {
 	}
 }
 
+func ToLogLevel(s string) LogLevel {
+	switch s {
+	case "debud":
```
Comment (jmank88) [inline core/web/resolver/log.go:31]:
```suggestion
	case "debug":
```

**Comment by cl-homu** [comment]
:umbrella: The latest upstream changes (presumably 9e6de7a599928469160c27e883c6bcecfe860b47) made this pull request unmergeable. Please [resolve the merge conflicts](https://rustc-dev-guide.rust-lang.org/git.html#rebasing-and-conflicts).

File: core/internal/testutils/configtest/general_config.go [hunk 2]
```diff
@@ -46,6 +47,7 @@ type GeneralConfigOverrides struct {
 	Dialect                                   dialects.DialectName
 	EVMDisabled                               null.Bool
 	EthereumDisabled                          null.Bool
+	EthereumURL                               null.String
```
Comment (jkongie) [inline core/internal/testutils/configtest/general_config.go:50]: Why was this added?

Comment (vyzaldysanchez) [reply core/internal/testutils/configtest/general_config.go:50]: I added as part of the overrides cause i wanted to add the ability to manually set the ETH URL manually, instead of depending on whatever value was being set behind the scenes. I could delete it if fine not having this as part of the overrides.

**Review by jkongie — APPROVED** [review]

**Review by jkongie — DISMISSED** [review]

**Comment by cl-homu** [comment]
:umbrella: The latest upstream changes (presumably 52bec6fc8b6404c8f75cebe9af13f69ecb185f1b) made this pull request unmergeable. Please [resolve the merge conflicts](https://rustc-dev-guide.rust-lang.org/git.html#rebasing-and-conflicts).

**Review by jkongie — APPROVED** [review]
